core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -43,12 +46,6 @@
    */
   private static final LogService LOGGER =
       LogServiceFactory.getLogService(FileFactory.class.getName());
-  private static Configuration configuration = null;
-
-  static {
-    configuration = new Configuration();
-    configuration.addResource(new Path("../core-default.xml"));
```

Comment:
This code can be kept, if configuration is not present in thread then you can get from here